webapp/components/Header/header.less

```diff
@@ -9,6 +9,9 @@
   background: white;
   color: @text-mute;
   padding: 0 20px;
+  &.pan-european {
+    grid-template-columns: 27px calc(@ui-app-navigation-width - 46px) 1px auto auto;
+  }
```

Comment:
why is this different for pan European ?

Reply:
Added more elements, needed one more column

Reply:
First one is the home icon, second one is the language switcher, the horizontal line, etc

Reply:
Otherwise, the "Pan european" text would collide with the logos of the right hand side

Reply:
I thought Home button and language switcher could fit into the first column?

Reply:
I can wrap them in a div